app/controllers/api/v1/comments_controller.rb

```diff
@@ -0,0 +1,32 @@
+class Api::V1::CommentsController < ApplicationController
+  load_and_authorize_resource
+  skip_before_action :authenticate_user!
+  skip_before_action :verify_authenticity_token
+  after_action :hard_logout
+  before_action :authenticate_request
+  skip_authorization_check only: [:create]
+
+  def index
+    @comments = User.find(params[:user_id]).posts.find(params[:post_id]).comments
+    render json: { success: true, data: { comments: @comments } }
+  end
+
+  def create
+    authorize! :create, @comment
+    @current_user = User.where(email: session[:user]['email']).first
+    @post = User.find(params[:user_id]).posts.find(params[:post_id])
+    @comment = @post.comments.new(text: comment_params[:text], user: @current_user)
+
+    if @comment.save
+      render json: { success: true, data: { comment: @comment } }, status: :created
+    else
+      render json: { success: false, errors: @comment.errors.full_messages }, status: :unprocessable_entity
+    end
+  end
```

Comment:
- You have done well so far, but I have this error when trying to create a comment under a particular post. Please fix this 🙏 
![wefwefefwef](https://github.com/ishimwepati/Blog-Rails/assets/73607512/0c724781-e1f1-4800-8f37-50187b82ea4c)
- According to the project requirements, the owner of the comment should be the user creating the comment. Please ensure you have properly configured your authentication for your api-endpoints. Good job using the `jwt` gem, but, optionally, I advise you consider looking into using [devise-jwt](https://enlear.academy/how-to-create-a-rails-6-api-with-devise-jwt-46fa35085e85). Please fix this 👍 👍

Reply:
Thanks, I have seen the error, the actual code I pushed them on an other Branch, 
But now everything is working fine !